products/jhipster.md

```diff
@@ -0,0 +1,72 @@
+---
+title: JHipster
+category: app
+versionCommand: yo jhipster
+permalink: /jhipster
+releasePolicyLink: https://www.jhipster.tech/releases/
+changelogTemplate: "https://github.com/jhipster/generator-jhipster/tag/__LATEST__"
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Supported
+#iconSlug: jhipster , WIP see https://github.com/jhipster/generator-jhipster/issues/20533
+
+auto:
+-   npm: https://www.npmjs.com/package/generator-jhipster
+
+releases:
+-   releaseCycle: "7"
+    eol: false
+    support: true
+    latest: "7.9.3"
+    latestReleaseDate: 2022-09-03
+    releaseDate: 2021-03-21
+
+-   releaseCycle: "6"
+    eol: 2021-03-21
+    latest: "6.10.5"
+    latestReleaseDate: 2020-11-07
+    releaseDate: 2019-05-10
```

Comment:
According to https://www.jhipster.tech/releases/, 6.0.0 was released on 2019-05-02.

Reply:
Fixed.